yal/utils.py

```diff
@@ -77,3 +79,25 @@ def replace_persona_fields(text, metadata={}):
         else:
             text = text.replace(f"[{key}]", PERSONA_DEFAULTS[key])
     return text
+
+
+def get_keywords(name):
+    keywords = []
+    with open(f"yal/keywords/{name}.csv", mode="r") as keyword_file:
+        csvreader = reader(keyword_file)
+        next(csvreader)
+        for row in csvreader:
+            keywords.extend([x.lower() for x in row if x != ""])
+    return keywords
+
+
+def check_keyword(keyword, keyword_list):
+    if process.extract(
+        keyword,
+        keyword_list,
+        scorer=fuzz.ratio,
+        score_cutoff=75,
+    ):
+        return True
+    else:
+        return False
```

Comment:
We can either do `return bool(process.extract(...))` or just `return process.extract(...)` to simplify this https://adamj.eu/tech/2020/01/17/simplify-your-ifs-that-return-booleans/